### DISCUSSION TITLE
Activated Tesla custom add-on

### DISCUSSION THREAD

**Discussion by Leytronix** [alandtse/tesla#184] [Q&A]
I have installed the Tesla custom addon in HACS and also generated the Tesla token with the App. But where do I enter the token in order to activate the addon?

**Comment by alandtse** [comment]
Add the integration n the integration page. You may have to hard refresh your browser.

**Comment by Leytronix** [comment]
Hi Alan,

 

Yes, I already did that (see below) but where to enter the token?

 

Thanks,

 

Tjerk

 



 

Van: Alan Tse ***@***.***> 
Verzonden: donderdag 31 maart 2022 09:37
Aan: alandtse/tesla ***@***.***>
CC: Tjerk bij de Leij ***@***.***>; Author ***@***.***>
Onderwerp: Re: [alandtse/tesla] Activated Tesla custom add-on (Discussion #184)

 

Add the integration n the integration page. You may have to hard refresh your browser.

**Reply by alandtse** [reply]
Seems obvious to me...
![image](https://user-images.githubusercontent.com/7086117/161004477-92fc43b9-8b4f-4659-a9d7-eda04451de02.png)

**Comment by Leytronix** [comment]
How to access this configuration window?

 

Van: Alan Tse ***@***.***> 
Verzonden: donderdag 31 maart 2022 09:48
Aan: alandtse/tesla ***@***.***>
CC: Tjerk bij de Leij ***@***.***>; Author ***@***.***>
Onderwerp: Re: [alandtse/tesla] Activated Tesla custom add-on (Discussion #184)

 

Seems obvious to me...
 <https://user-images.githubusercontent.com/7086117/161004477-92fc43b9-8b4f-4659-a9d7-eda04451de02.png>
